dear_petition/petition/models.py

```diff
@@ -168,12 +168,15 @@ def refresh_record_from_data(self):
                 disposition_method=offenses.get("Disposition Method", ""),
             )
```

Comment:
If there are more `Offense`s for this `CIPRSRecord` in the database, should we delete them here too?

Reply:
yep

Reply:
done